javascript/0034-find-first-and-last-position-of-element-in-sorted-array.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Binary Search
+ * https://leetcode.com/problems/find-first-and-last-position-of-element-in-sorted-array/
+ * Time O(log(n)) | Space O(1)
+ * @param {number[]} nums
+ * @param {number} target
+ * @return {number[]}
+ */
+var searchRange = function(nums, target) {
+    
+    const result = [];
+
+    result.push(binarySearch(true));
+    result.push(binarySearch(false));
+
+    function binarySearch(leftBias) {
```

Comment:
Suggestion: name booleans that prompt a question

```js
isLeftBias
```

Reply:
Just a kind reminder that I updated the code as suggested.